src/abstractdataframe/selection.jl

```diff
@@ -125,11 +125,20 @@ end
 
 function select_transform!(nc::Pair{Int, Pair{ColRename, Symbol}},
                            df::AbstractDataFrame, newdf::DataFrame,
-                           transformed_cols::Dict{Symbol, Any}, copycols::Bool)
+                           transformed_cols::Dict{Symbol, Any}, copycols::Bool,
+                           allow_resizing_newdf::Base.RefValue{Bool})
```

Comment:
`Ref` is the supertype so it works:
```suggestion
                           allow_resizing_newdf::RefBool})
```